crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -257,7 +257,7 @@ def upgrade_controller_nodes
         NodeObject.find(
           "state:crowbar_upgrade AND (roles:database-server OR roles:rabbitmq-server)"
         ).each do |db_node|
-          cmd = "LANG=C crm resource status ms-drbd-{postgresql,rabbitmq}\
+          cmd = "LANG=C crm resource status ms-drbd-postgresql\
```

Comment:
@rhafer what about this - take only one resource that is actually running there:

`LANG=C crm resource status ms-drbd-{postgresql,rabbitmq} | sort | head -n 2 | grep \\$(hostname)`